### PR TITLE
V0.0.38

### PR DESCRIPTION
**Pull request by tony-go**
Hi 👋 

This pull request announces the version `0.0.38`

Added:
* Run init before run command if no config file exists #166 

Updated:
* Bump a few packages: `templ`, `ws`, `url-parse`, `prismjs`, `axios` & `post-css`.

### CODE REVIEW

**Comment by tony-go** [comment]
Release candidate: `0.0.38-rc.0`or `next` ready!

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (olivierodo) [inline .vscode/settings.json:1]: Necessary ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.37] - 2021-12-08
```
Comment (olivierodo) [inline CHANGELOG.md:10]: missing 0.0.38 ?

File: bin/restqa [hunk 3]
```diff
@@ -1,115 +1,30 @@
...
-const logger = require('../src/utils/logger')
-const updateNotifier = require('update-notifier')
-const path = require('path')
-const Telemetry = require('../src/utils/telemetry')
+const pkg = require("../package.json");
+const updateNotifier = require("update-notifier");
 
-const DEFAULT_CONFIG = process.env.RESTQA_CONFIG || path.resolve(process.cwd(), '.restqa.yml')
+const logger = require("../src/utils/logger");
+const Telemetry = require("../src/utils/telemetry");
```
Comment (olivierodo) [inline bin/restqa:7]: Can we reuse the same Telemetry instance from the `bin/program.js`

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@restqa/restqa",
-  "version": "0.0.36",
+  "version": "0.0.37",
```
Comment (olivierodo) [inline package.json:3]: 0.0.38 ?

**Review by olivierodo — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (tony-go) [reply .vscode/settings.json:1]: Yeap' it helps really ... for vscode users 😜

File: CHANGELOG.md (hunk 2 above)
Comment (tony-go) [reply CHANGELOG.md:10]: My workflow is mainly to let changelog and bump version when all is ready ^^

**Comment by tony-go** [comment]
🎉 `@restqa/restqa@0.0.38-rc.1` ready for testing!

**Review by olivierodo — APPROVED** [review]

**Comment by tony-go** [comment]
LGTM 🚀

**Pull request merged**